cocotb/bus.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# Copyright (c) 2013 Potential Ventures Ltd
+# Copyright (c) 2013, 2019 Potential Ventures Ltd
```

Comment:
Please don't update the copyright year. I've opened #804 to see how we can fix the headers in the longer run.